tests/kernel/ltp_setup_networking.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This module installs and setup services and other things needed for
+# networking part of the LTP (Linux Test Project).
+# Maintainer: Petr Vorel <pvorel@suse.cz>
+
+use 5.018;
+use warnings;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+
+sub install_services {
+    my $self = shift;
```

Comment:
Thanks for your review. Sure I'll remove $self and improve commit message.

Just about signing commits. Does it make any harm? It's a required on many projects (https://www.kernel.org/doc/Documentation/SubmittingPatches "11) Sign your work"). Of course kernel is a different project, but all projects I know require it. Well they mailing list managed, not github managed. So if you require it, I'll remove it.